Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -171,6 +171,9 @@ CSI-Addons supports the following operations:
   * [Annotating PersistentVolumeClaims](https://github.com/csi-addons/kubernetes-csi-addons/blob/v0.3.0/docs/reclaimspace.md#annotating-perstentvolumeclaims)
 * Network Fencing
   * [Creating a NetworkFence](https://github.com/csi-addons/kubernetes-csi-addons/blob/v0.3.0/docs/networkfence.md)
+* Volume Replication
+  * [Creating VolumeReplicationClass](https://github.com/csi-addons/kubernetes-csi-addons/blob/main/docs/volumereplicationclass.md)
```

Comment:
These URLs use main, but the other URLs on above lines have v0.3.0. Is there a `latest` tag (or similar) instead? At least we should make them consistent.

Reply:
Above I see some other links in this PR to v0.5.0. Is that the latest version?

Reply:
oh yeah, we have the latest version v0.5.0 out, forgot to update it for reclaimspace and network fence. Will do that. Thanks @travisn for mentioning it.